src/contribution-guide/contributing.md

```diff
@@ -0,0 +1,24 @@
+# Contribute to Kubewarden docs
+
+_Kubewarden welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to Kubewarden documentation by
+
+- Improving existing content
+- Creating new content
+- (_Upcoming_) Helping with translation of the documentation
+
+## Getting Started
```

Comment:
This is super minor, but in all other headers, you have followed the style of capping the first and proper words and leaving the rest lowercase. As such, this can be **Getting started**.